services/src/main/resources/config-spring-geonetwork.xml

```diff
@@ -76,21 +77,28 @@
             <value>Conformance Class 2: INSPIRE data sets and data set series interoperability metadata.</value>
           </array>
         </entry>
-        <entry key="TG version 2.0 - Spatial data service">
+        <entry key="TG version 2.0 - Network services">
           <array>
             <value>Common Requirements for ISO/TC 19139:2007 based INSPIRE metadata records.</value>
-            <value>Conformance Class 1: INSPIRE data sets and data set series baseline metadata.</value>
-            <value>Conformance Class 2: INSPIRE data sets and data set series interoperability metadata.</value>
+            <!--<value>Conformance Class 1: INSPIRE data sets and data set series baseline metadata.</value>
+            <value>Conformance Class 2: INSPIRE data sets and data set series interoperability metadata.</value>-->
             <value>Conformance Class 3: INSPIRE Spatial Data Service baseline metadata.</value>
-            <!--<value>Conformance Class 4: INSPIRE Network Services metadata.</value>
-            <value>Conformance Class 5: INSPIRE Invocable Spatial Data Services metadata.</value>
+            <value>Conformance Class 4: INSPIRE Network Services metadata.</value>
+            <!--<value>Conformance Class 5: INSPIRE Invocable Spatial Data Services metadata.</value>
             <value>Conformance Class 6: INSPIRE Interoperable Spatial Data Services metadata.</value>
             <value>Conformance Class 7: INSPIRE Harmonised Spatial Data Services metadata.</value>-->
           </array>
         </entry>
       </map>
     </property>
+    <property name="testsuitesConditions">
```

Comment:
Would make sense to have on config per schema. eg. ISO19115-3 xpath will use different namespaces